app/electron/main.ts

```diff
@@ -78,6 +82,20 @@ const args = yargs(hideBin(process.argv))
   .help()
   .parseSync();
 
+const listPlugins = args['list-plugins'] === true;
+
+if (listPlugins) {
+  try {
+    const backendPath = path.join(process.resourcesPath, 'headlamp-server');
+    const stdout = execSync(`${backendPath} --list-plugins`);
+    console.log(stdout.toString().trim());
```

Comment:
I noticed a discrepancy in formatting between the outputs from the app and from the backend, turns out the fix is using `process.stdout.write` rather than `console.log`